runtime_builders/builder_util.py

```diff
@@ -57,10 +63,57 @@ def get_file_from_gcs(gcs_file, temp_file):
         return False
 
 
-def write_manifest_file(manifest):
-    manifest_contents = yaml.round_trip_dump(manifest,
-                                             default_flow_style=False)
-    write_to_gcs(MANIFEST_FILE, SCHEMA_LINE + manifest_contents)
+def verify_manifest(manifest):
```

Comment:
If you need additional inspiration you can look at runtime_builders.py:361 where I do similar parsing